src/api/data/charts.go

```diff
@@ -8,32 +8,47 @@ import (
 	"github.com/helm/monocular/src/api/pkg/swagger/models"
 )
 
-// GetChart gets the chart associated with the passed-in repo+name
-func GetChart(repo, name string) (models.Resource, error) {
-	// TODO implement actual "get chart" business logic
+// Charts is an interface for managing chart data
+type Charts interface {
+	// will have a GetChart method to retrieve a particular chart from a repo
+	GetChart(repo, name string) (models.Resource, error)
+	// will have a GetAllFromRepo method to retrieve all charts from a repo
+	GetAllFromRepo(repo string) ([]*models.Resource, error)
+	// will have a GetAll method to retrieve all charts from all repos
+	GetAll() ([]*models.Resource, error)
+}
+
+// mockCharts fulfills the Charts interface
+type mockCharts struct{}
```

Comment:
it's a bit academic, but for now there's now exportable entrance use-case outside of the constructor, so we'll keep the interface struct implementation private